src/esm_runscripts/sim_objects.py

```diff
@@ -165,11 +165,6 @@ def prepcompute(self):
     def viz(self):
         """
         Starts the Viz job.
-
-        Parameters
-        ----------
-        kill_after_submit: bool
-            Default ``True``. If set, the entire Python instance is killed with ``sys.exit()``.
```

Comment:
Incorrect docs are even worse than missing docs. This method doesn't get any parameters...